test/integration/http.js

```diff
@@ -393,15 +393,79 @@ describe('HTTP APIs', function () {
         .expect(hasHeader('acl', 'baz.ttl' + suffixAcl))
         .expect(201, done)
     })
-    it('should return 409 code when trying to put to a container',
-      function (done) {
-        server.put('/')
-          .expect(409, done)
-      }
-    )
-    // Cleanup
-    after(function () {
-      rm('/foo/')
+
+    describe('PUT and containers', () => {
+      const containerMeta = fs.readFileSync(path.join(__dirname,
+        '../resources/sampleContainer/post2.ttl'),
+        { 'encoding': 'utf8' })
+
+      before(() => {
+        rm('/foo/')
+      })
+
+      after(() => {
+        rm('/foo/')
+      })
```

Comment:
I understand cleanup is necessary, but I've never had to do it both before and after a test is run.
Either should suffice.